reporter-influxdb/src/test/java/io/ultrabrew/metrics/reporters/influxdb/InfluxDBClientTest.java

```diff
@@ -147,6 +147,33 @@ public void testWriteSplitting() throws IOException {
     }};
   }
 
+  @Test
+  public void testNullStrings() throws IOException {
+    new Expectations() {{
+      httpClient.execute((HttpUriRequest) any);
+      result = closeableHttpResponse;
+      closeableHttpResponse.getEntity();
+      result = new BasicHttpEntity();
+      closeableHttpResponse.getStatusLine();
+      result = statusLine;
+      statusLine.getStatusCode();
+      result = 200;
+    }};
+    client.write(null, new String[] { "host", "web01" }, new String[] { "temp", "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { null, "web01" }, new String[] { "temp", "80" }, 1534055562000000003L);
+    // one good one in the middle
+    client.write("cpu_load_short", new String[] { "host", "web01" }, new String[] { "temp", "80" }, 1534055562000000003L);
+    // this is ok too
+    client.write("cpu_load_short", new String[] { "host", null }, new String[] { "temp", "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { "host", "web01" }, new String[] { null, "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { "host", "web01" }, new String[] { "temp", null }, 1534055562000000003L);
+    client.flush();
+    new Verifications() {{
+      httpClient.execute((HttpUriRequest) any);
```

Comment:
Assert something about the payload sent here, to verify that it contains the expected data?

Reply:
yeah I just have to dig into Mockit and figure out how to parse the payload.

Reply:
Something along these lines should work:
```java
    new Verifications() {{
      HttpPost request;
      httpClient.execute(request = withCapture());
      assertThat(request.getEntity().someProperty(), is(someExpectedValue));
}};
```